packages/commonwealth/client/scripts/controllers/server/sessions.ts

```diff
@@ -123,6 +123,9 @@ async function sign(
         address: lookupAddress,
       });
       if (!savedSessionMessage) {
+        if (!app.config.enforceSessionKeys) {
```

Comment:
lets do early return at the start of function

Reply:
we should still try to sign messages if the user has a session though

should i just inline the if statements?